packages/cli/src/checks.ts

```diff
@@ -72,16 +63,24 @@ export const runChecks = async (
   );
 
   const total = rulesToRunAccordingToLevel.length + 1;
+  let current = 0;
 
-  const rulesProgressBar = progressBar.create(
-    total,
-    0,
-    {},
-    { format: 'Rules:  {bar} {percentage}% | ETA: {eta}s | {value}/{total}' },
-  );
+  const rulesSpinner = new Spinner({
+    text: 'Processing rules... %s',
```

Comment:
`Processing rule ${current}/${total}... %s`

maybe export it in a function bc its the same as in 'decreaseRemaining'

You can remove the other options they are default

Reply:
also, try to use chalk around `%s` to color the spinner